<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimulatorEmulatedMonitorDialog</class>
 <widget class="QDialog" name="CSimulatorEmulatedMonitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Emulated simulator monitor</string>
  </property>
  <layout class="QVBoxLayout" name="vl_Settings">
   <item>
    <widget class="QFrame" name="fr_LedHeader">
     <layout class="QHBoxLayout" name="hl_LedHeader">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_Receiving">
        <property name="text">
         <string>Receiving</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="swift::gui::CLedWidget" name="led_Receiving" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="lbl_Sending">
        <property name="text">
         <string>Sending</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="swift::gui::CLedWidget" name="led_Sending" native="true"/>
      </item>
      <item>
       <spacer name="hs_LedHeader">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tw_SwiftMonitorDialog">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_AircraftSituation">
      <attribute name="title">
       <string>Situation</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_Situation">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="swift::gui::editors::CSituationForm" name="editor_Situation">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_ComParts">
      <attribute name="title">
       <string>COM and Parts</string>
      </attribute>
      <layout class="QGridLayout" name="gl_ComParts">
       <item row="2" column="0">
        <spacer name="vs_Com">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="gb_Com">
         <property name="title">
          <string>COM</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="swift::gui::editors::CCockpitComForm" name="editor_Com">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>75</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="QGroupBox" name="gb_Parts">
         <property name="title">
          <string>Parts</string>
         </property>
         <layout class="QVBoxLayout" name="vl_GroupBoxParts">
          <item>
           <widget class="swift::gui::editors::CAircraftPartsForm" name="editor_AircraftParts">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>125</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="swift::gui::components::CCockpitComTransmissionComponent" name="comp_ComTransmissions">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Simulator">
      <attribute name="title">
       <string>Simulator</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_Simulator">
       <item>
        <widget class="QGroupBox" name="gb_General">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QFormLayout" name="fl_General">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_SimulatorPlugin">
            <property name="text">
             <string>Simulator plugin:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_SimulatorPlugin">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_Simulator">
            <property name="text">
             <string>Simulator:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_Simulator">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbl_Status">
            <property name="text">
             <string>Status</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QFrame" name="fr_Status">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="hl_SimulatorStatus">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="cb_Connected">
               <property name="text">
                <string>simulator connected?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_Simulating">
               <property name="text">
                <string>simulator simulating?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_Paused">
               <property name="text">
                <string>simulator paused?</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_Signals">
         <property name="title">
          <string>Signals</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="wi_AddedFailed" native="true">
            <layout class="QHBoxLayout" name="hl_AddedFailed">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="swift::gui::components::CCallsignCompleter" name="comp_CallsignCompleter">
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_Failover">
               <property name="text">
                <string>failover</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="hs_AddedFailed">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="QPushButton" name="pb_EmitAddedFailed">
               <property name="text">
                <string>emit</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_Interpolator">
         <property name="title">
          <string>Interpolator</string>
         </property>
         <layout class="QGridLayout" name="gl_Interpolator">
          <item row="1" column="6">
           <widget class="QPushButton" name="pb_InterpolatorStartLog">
            <property name="text">
             <string> start log. </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pb_InterpolatorClearLog">
            <property name="text">
             <string> clear log </string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pb_InterpolatorStopLog">
            <property name="text">
             <string> stop logging </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pb_InterpolatorWriteLog">
            <property name="text">
             <string> write log.file </string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pb_InterpolatorShowLogs">
            <property name="text">
             <string> log.files </string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="swift::gui::components::CCallsignCompleter" name="comp_LogInterpolatorCallsign">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pb_InterpolatorFetch">
            <property name="text">
             <string>fetch</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="le_InterpolatorTimeMs">
            <property name="text">
             <string>40</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
            <property name="placeholderText">
             <string>fetch time ms</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="swift::gui::CLedWidget" name="led_Fetching" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_Messages">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Messages (Text/Status messages)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSplitter" name="splitter_Messages">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPlainTextEdit" name="pte_TextMessages">
             <property name="whatsThis">
              <string>text messages</string>
             </property>
             <property name="documentTitle">
              <string>Received text messages</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>text messages</string>
             </property>
            </widget>
            <widget class="QPlainTextEdit" name="pte_StatusMessages">
             <property name="whatsThis">
              <string>status messages</string>
             </property>
             <property name="documentTitle">
              <string>Received status messages</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>status messages</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_LogMessages">
      <attribute name="title">
       <string>API/Log information</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_LogInformation">
       <item>
        <widget class="QGroupBox" name="gb_Statistics">
         <property name="title">
          <string>Statistics</string>
         </property>
         <layout class="QGridLayout" name="gl_Statistics">
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_SituationAdded">
            <property name="text">
             <string>Situation added:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_SituationAdded">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="le_PartsAdded">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="lbl_PartsAdded">
            <property name="text">
             <string>Parts added:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="lbl_AircraftRendered">
            <property name="text">
             <string>Aircraft rendered:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QLineEdit" name="le_AircraftRendered">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_PhysicallyAddedAircraft">
            <property name="text">
             <string>Phy.added aircraft:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_PhysicallyAddedAircraft">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="le_PhysicallyRemovedAircraft">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <widget class="QPushButton" name="pb_ResetStatistics">
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="lbl_PhysicallyRemovedAircraft">
            <property name="text">
             <string>Phy.removed aircraft:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="8">
           <widget class="QLineEdit" name="le_PartsEnabledAircraft">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="lbl_PartsEnabledAircraft">
            <property name="text">
             <string>Parts enabled:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_AutoPublish">
         <property name="title">
          <string>Auto publish</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="pb_AddAutoPublishData">
            <property name="text">
             <string>add</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_LogMessages">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Simulator API calls</string>
         </property>
         <layout class="QVBoxLayout" name="vl_GroupBoxLogMessages">
          <item>
           <widget class="QScrollArea" name="sa_LogMessages">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="sa_LogMEssagesWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>634</width>
               <height>384</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="vl_LogMessagesScrollArea">
              <item>
               <widget class="swift::gui::components::CStatusMessagesDetail" name="comp_LogComponent">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>325</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Settings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="gb_Settings">
         <property name="title">
          <string>Simulator settings</string>
         </property>
         <layout class="QVBoxLayout" name="vl_GroupBoxSettings">
          <item>
           <widget class="swift::gui::components::CSettingsSwiftPlugin" name="comp_Settings"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_SwiftMonitorDialog">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CSettingsSwiftPlugin</class>
   <extends>QFrame</extends>
   <header>gui/components/settingsswiftplugin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CStatusMessagesDetail</class>
   <extends>QFrame</extends>
   <header>gui/components/statusmessagesdetail.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::editors::CSituationForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/situationform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::editors::CAircraftPartsForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/aircraftpartsform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::editors::CCockpitComForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/cockpitcomform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::CLedWidget</class>
   <extends>QWidget</extends>
   <header>gui/led.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CCallsignCompleter</class>
   <extends>QFrame</extends>
   <header>gui/components/callsigncompleter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CCockpitComTransmissionComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/cockpitcomtransmissioncomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tw_SwiftMonitorDialog</tabstop>
  <tabstop>le_SimulatorPlugin</tabstop>
  <tabstop>le_Simulator</tabstop>
  <tabstop>cb_Connected</tabstop>
  <tabstop>cb_Simulating</tabstop>
  <tabstop>cb_Paused</tabstop>
  <tabstop>cb_Failover</tabstop>
  <tabstop>pb_EmitAddedFailed</tabstop>
  <tabstop>le_InterpolatorTimeMs</tabstop>
  <tabstop>pb_InterpolatorFetch</tabstop>
  <tabstop>pb_InterpolatorClearLog</tabstop>
  <tabstop>pb_InterpolatorWriteLog</tabstop>
  <tabstop>pb_InterpolatorStopLog</tabstop>
  <tabstop>pb_InterpolatorShowLogs</tabstop>
  <tabstop>pb_InterpolatorStartLog</tabstop>
  <tabstop>pte_TextMessages</tabstop>
  <tabstop>pte_StatusMessages</tabstop>
  <tabstop>le_PhysicallyAddedAircraft</tabstop>
  <tabstop>le_PhysicallyRemovedAircraft</tabstop>
  <tabstop>le_AircraftRendered</tabstop>
  <tabstop>pb_ResetStatistics</tabstop>
  <tabstop>le_SituationAdded</tabstop>
  <tabstop>le_PartsAdded</tabstop>
  <tabstop>le_PartsEnabledAircraft</tabstop>
  <tabstop>pb_AddAutoPublishData</tabstop>
  <tabstop>sa_LogMessages</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bb_SwiftMonitorDialog</sender>
   <signal>accepted()</signal>
   <receiver>CSimulatorEmulatedMonitorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_SwiftMonitorDialog</sender>
   <signal>rejected()</signal>
   <receiver>CSimulatorEmulatedMonitorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
